lib/genome/importers/dtc/dtc.rb

```diff
@@ -0,0 +1,70 @@
+require 'genome/online_updater'
+
+module Genome; module Importers; module Dtc;
+class DtcImporter < Genome::OnlineUpdater
+  attr_reader :file_path, :source
+
+  def initialize(file_path)
+    @file_path = file_path
+  end
+
+  def get_version
+    source_db_version = Date.today.strftime("%d-%B-%Y")
+    @new_version = source_db_version
+  end
+
+  def import
+    remove_existing_source
+    create_new_source
+    create_interaction_claims
+  end
+
+  private
+  def remove_existing_source
+    Utils::Database.delete_source('DTC')
+  end
+
+  def create_new_source
+    @source ||= DataModel::Source.create(
+        {
+            base_url: 'https://drugtargetcommons.fimm.fi/',
+            site_url: 'https://drugtargetcommons.fimm.fi/',
+            citation: 'Drug Target Commons 2.0: a community platform for systematic analysis of drug–target interaction profiles. Tanoli Z, Alam Z, Vähä-Koskela M, Malyutina A, Jaiswal A, Tang J, Wennerberg K, Aittokallio T. Database. 2018.',
+            source_db_version:  get_version,
+            source_type_id: DataModel::SourceType.INTERACTION,
+            source_db_name: 'DTC',
+            full_name: 'Drug Target Commons',
+            license: 'Creative Commons Attribution-NonCommercial 3.0 (BY-NC)',
+            license_link: 'https://academic.oup.com/database/article/doi/10.1093/database/bay083/5096727',
+        }
+    )
+  end
+
+  def create_interaction_claims
+    CSV.foreach(file_path, :headers => true) do |row|
+      drug_id = row['compound_id']
+      drug_name = row['compound_name']
+      gene_name = row['gene_names']
+      pmid = row['pubmed_id']
+      mechanism = row['ep_action_mode']
+      unless drug_name.nil? || gene_name.nil?
+        drug_claim = create_drug_claim(gene_name, gene_name, 'DTC Drug Name')
```

Comment:
You are using the `gene_name` here instead of the `drug_name`

Reply:
I fixed this